components/navbar/StatsDropdown.vue

```diff
@@ -1,83 +1,125 @@
+            data-cy="global-offers"
+            :to="offersUrl"
+            tag="nuxt-link">
+            {{ $t('navbar.globalOffers') }}
+          </b-navbar-item>
+          <b-navbar-item data-cy="offers-stats" :to="statsUrl" tag="nuxt-link">
+            {{ $t('navbar.offerStats') }}
+          </b-navbar-item>
+          <b-navbar-item
+            data-cy="series-insight"
+            to="/series-insight"
+            tag="nuxt-link">
+            {{ $t('series.label') }}
+          </b-navbar-item>
+        </template>
+        <template v-if="chain === 'rmrk'">
+          <b-navbar-item data-cy="spotlight" to="/spotlight" tag="nuxt-link">
+            {{ $t('spotlight.page') }}
+          </b-navbar-item>
+          <b-navbar-item
+            data-cy="series-insight"
+            to="/series-insight"
+            tag="nuxt-link">
+            {{ $t('series.label') }}
+          </b-navbar-item>
+          <b-navbar-item data-cy="sales" to="/sales" tag="nuxt-link">
+            {{ $t('sales.page') }}
+          </b-navbar-item>
+          <b-navbar-item data-cy="hot" to="/hot" tag="nuxt-link">
+            {{ $t('hot.label') }}
+          </b-navbar-item>
+        </template>
       </template>
-      <b-dropdown-item has-link aria-role="menu-item">
-        <nuxt-link data-cy="spotlight" to="/spotlight">
-          {{ $t('spotlight.page') }}
-        </nuxt-link>
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="series-insight" to="/series-insight">
-          Series</nuxt-link
-        >
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="sales" to="/sales"> Sales</nuxt-link>
-      </b-dropdown-item>
-      <b-dropdown-item has-link>
-        <nuxt-link data-cy="hot" to="/hot"> Hot</nuxt-link>
-      </b-dropdown-item>
-    </b-dropdown>
+    </MobileExpandableSection>
   </div>
 </template>
-<script lang="ts">
-import { Component, Prop, mixins } from 'nuxt-property-decorator'
-import { getChainTestList } from '~/utils/constants'
-import PrefixMixin from '@/utils/mixins/prefixMixin'
-import AuthMixin from '~~/utils/mixins/authMixin'
+<script lang="ts" setup>
+import { isMobileDevice } from '~~/utils/extension'
 
-@Component({})
-export default class NavbarCreate extends mixins(PrefixMixin, AuthMixin) {
-  @Prop({ type: String }) chain!: string
+const MobileExpandableSection = defineAsyncComponent(
+  () => import('@/components/navbar/MobileExpandableSection.vue')
+)
+const { accountId } = useAuth()
 
-  get options() {
-    const availableUrlPrefixes = this.$store.getters['availableUrlPrefixes']
+const { urlPrefix } = usePrefix()
+const props = defineProps<{
+  chain?: string
+}>()
 
-    if (!this.$config.dev) {
-      return availableUrlPrefixes.filter(
-        (urlPrefix) => !getChainTestList().includes(urlPrefix.value as string)
-      )
-    }
-    return availableUrlPrefixes
-  }
-}
+const isMobile = window.innerWidth < 1024 ? true : isMobileDevice
```

Comment:
Why not `const isMobile = isMobileDevice`

Reply:
It was actually done like this in one of the other components, but I used const isMobile = isMobileDevice this now